develop-guide/writetrigger/writetrigger.rst

```diff
@@ -587,60 +587,179 @@ Here is the list of all available actions. These actions are to be placed at thi
 
 .. code-block:: YAML
 
-  # Name of the testcase
-  Testcase1:
-      # Location of the blitz trigger
-      source:
-        pkg: genie.libs.sdk
-        class: triggers.blitz.blitz.Blitz
+    # Name of the testcase
+    Testcase1:
 
-      # Field containing all the sections
-      test_sections:
-
-        # Section name - Can be any name, it will show as the first section 
-        # of the testcase
-        - apply_configuration:
-            - >>>> <ACTION> <<<< 
-            - >>>> <ACTION> <<<< 
-            - >>>> <ACTION> <<<< 
-            - >>>> <ACTION> <<<< 
-        - section_two::
-            - >>>> <ACTION> <<<< 
-            - >>>> <ACTION> <<<< 
+        # Location of the blitz trigger
+        source:
+            pkg: genie.libs.sdk
+            class: triggers.blitz.blitz.Blitz
+
+        # Field containing all the sections
+        test_sections:
+
+            # Section name - Can be any name, it will show as the first section
+            # of the testcase
+            - apply_configuration:
+                devices:
+
+                    # Device to use
+                    - uut:

```

Comment:
This is using old syntax please update to new, without device

Reply:
Say that all the actions are defined below, right now, its confusing to know what to put here